openedx/core/djangoapps/programs/utils.py

```diff
@@ -373,6 +373,7 @@ def _attach_run_mode_certificate_url(self, run_mode):
         certificate_data = certificate_api.certificate_downloadable_status(self.user, self.course_key)
         certificate_uuid = certificate_data.get('uuid')
         run_mode['certificate_url'] = certificate_api.get_certificate_url(
+            user_id=self.user.id,
```

Comment:
Add a comment about why we had to add this (e.g., allows us to fall back to PDF certificates if web certificates are not configured).